src/cli/index.ts

```diff
@@ -42,27 +41,27 @@ const signOptions = {
 
 async function sign(artifactPath: string) {
   const buffer = fs.readFileSync(artifactPath);
-  const bundle = await sigstore.sign(buffer, signOptions);
+  const bundle: any = await sigstore.sign(buffer, signOptions); //eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
😞

Reply:
Yeah, don't like this at all. If we're gonna go this route, I'd want to define a custom interface which matches the serialized form of the Bundle (see my comments below).